packages/core/src/app/ui/form/Checklist.tsx

```diff
@@ -29,6 +30,12 @@ export const ChecklistContext = createContext<ChecklistContextProps | undefined>
 const Checklist: FunctionComponent<
     ChecklistProps & ConnectFormikProps<{ [key: string]: string }>
 > = ({ formik: { setFieldValue }, name, onSelect = noop, ...props }) => {
+    useEffect(() => {
```

Comment:
It looks like `PaymentMethodList` also uses `Checklist` component. Can you double check nothing breaks?